### DISCUSSION TITLE
Hi, I am new to React Hook Form, Is there a way to clear my form fields after submitting the form?

### DISCUSSION THREAD

**Discussion by adenugatani** [react-hook-form/react-hook-form#12297] [Q&A] [answered]
Hi, I am new to React Hook Form, Is there a way to clear my form fields after submitting the form?

**Comment by wilsonadenuga** [comment, marked as answer]
Hello @tanitoluwam yes there is a way, react-hook-form comes with a reset helper that helps to reset form state, you can find more info [here](https://react-hook-form.com/docs/useform/reset)

**Reply by adenugatani** [reply]
Oh..... Thank you so much !
